Stack.c

```diff
@@ -0,0 +1,156 @@
+#include "Stack_config.h"
+#include <stdlib.h>
+#include <stdint.h>
```

Comment:
DONOT USE `stdint.h` you have to use out datatypes file and if not exists simply push it :